js_formatter.py

```diff
@@ -34,20 +32,13 @@ def _try_get_short_path(path):
 # crazyness to get jsbeautifier.unpackers to actually import
 # with sublime's weird hackery of the path and module loading
 add_lib_path(libs_path)
-add_lib_path(os.path.join(libs_path, "six-1.8.0"))
-add_lib_path(os.path.join(libs_path, "jsbeautifier"))
-add_lib_path(os.path.join(libs_path, "jsbeautifier", "jsbeautifier"))
```

Comment:
I've tested on Mac/Win with ST3. The thing is, I've kept `add_lib_path(libs_path)` and this `libs/` folder only contains modules (all its folders contain __init__.py). It's why it works perfectly.